config/dev.exs

```diff
@@ -0,0 +1,9 @@
+use Mix.Config
+
+config :paytm, Paytm.API.Wallet,
+  base_url: "https://pguat.paytm.com"
+
+config :paytm, Paytm.API.OAuth,
+  base_url: "https://accounts-uat.paytm.com",
+  client_id: "",
+  client_secret: ""
```

Comment:
I would move all this to env vars.
Then we could have everything on the `config/config.exs` but still be able to overwrite it per env if needed (`test` for instance`)
Just remember to remove the `@module_attrs` from the modules and extract it to methods to avoid issues with env vars.